docs/content/users/install/ddev-installation.md

```diff
@@ -46,6 +46,8 @@ Docker or an alternative is required before anything will work with DDEV. This i
   
     In the future you can update as usual, with `sudo apt update && sudo apt upgrade`.
 
+    If you previously used `install_ddev.sh` to install DDEV, you can just `sudo rm /usr/local/bin/ddev` to remove the previous version. If you previously used homebrew to install DDEV, you can just `brew unlink ddev` to get rid of the homebrew version.
```

Comment:
There are 3 artifacts which are copied by the script see https://github.com/drud/ddev/blob/master/scripts/install_ddev.sh#L12, I've removed them all while switching.

Reply:
You can just click the "Add a suggestion" button to improve the text :)

Reply:
Yes I know. Just wasn't sure if this was intended to omit these files ;)